Library/AppEnvironmentTests.swift

```diff
@@ -60,19 +60,23 @@ final class AppEnvironmentTests: XCTestCase {
     XCTAssertNil(AppEnvironment.current.currentUser)
 
     AppEnvironment.login(AccessTokenEnvelope(accessToken: "deadbeef", user: User.template))
+    AppEnvironment.current.userDefaults.analyticsIdentityData = KSRAnalyticsIdentityData(.template)
 
     XCTAssertEqual("deadbeef", AppEnvironment.current.apiService.oauthToken?.token)
     XCTAssertEqual(User.template, AppEnvironment.current.currentUser)
+    XCTAssertNotNil(AppEnvironment.current.userDefaults.analyticsIdentityData)
 
     AppEnvironment.updateCurrentUser(User.template)
```

Comment:
I may be missing something, but do we want to update the `User.template` here? I think it's the same some as the instantiated value. Just because I think we're testing that method for a change.